packages/loot-core/src/shared/environment.ts

```diff
@@ -1,8 +1,8 @@
-export function isPreviewEnvironment() {
+function isPreviewEnvironment() {
   return String(process.env.REACT_APP_NETLIFY) === 'true';
 }
 
-export function isDevelopmentEnvironment() {
+function isDevelopmentEnvironment() {
```

Comment:
IMO these should stay exported so that they can be used if needed.

Reply:
Makes sense, i'll add an eslint ignore to those.